hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/MultiSnapshotLocks.java

```diff
@@ -67,8 +67,10 @@ public synchronized OMLockDetails acquireLock(Collection<UUID> ids) throws OMExc
         lock.acquireReadLocks(resource, keys);
     if (omLockDetails.isLockAcquired()) {
       objectLocks.addAll(keys);
+      this.lockDetails = OMLockDetails.EMPTY_DETAILS_LOCK_ACQUIRED;
```

Comment:
Use a static variable to replace ThreadLock object.
two locks acquired by the same thread do not interefere.

Reply:
We cannot reuse the lockDetails across objects it has to be a object member